packages/atomic/cypress/integration/facets/date-facet/date-facet-assertions.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  convertDateFormatLabel,
+  convertDateToFacetValue,
+  dateField,
+  DateRange,
+  dateRanges,
+} from './date-facet-actions';
+import {convertFacetValueToAPIformat} from '../facet/facet-actions';
+import {dateFacetComponent, FacetSelector} from '../facet/facet-selectors';
+
+export function assertDateFacetValueOnUrl(
```

Comment:
the method is used only once without parameters, so I would remove them as they are not necessary